tests/sitemaps_tests/test_generic.py

```diff
@@ -6,6 +7,16 @@
 
 @override_settings(ABSOLUTE_URL_OVERRIDES={})
 class GenericViewsSitemapTests(SitemapTestsBase):
+    @classmethod
+    def setUpClass(cls):
+        super(GenericViewsSitemapTests, cls).setUpClass()
+        cls.generic_sitemap = GenericSitemap(
+            info_dict={
+                'queryset': TestModel.objects.all(),
+            },
+            priority=0.6,
+            changefreq='monthly'
```

Comment:
include a trailing comma so if more arguments are added later, this line doesn't have to be modified again